packages/o/opentabletdriver/files/attach-to-default-target.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service b/eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service
+index 06b69dd7..3960c968 100644
+--- a/eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service
++++ b/eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service
+@@ -1,7 +1,7 @@
+ [Unit]
+ Description=OpenTabletDriver Daemon
+-PartOf=graphical-session.target
+-After=graphical-session.target
++PartOf=default.target
```

Comment:
Why? As far as I can tell this would make it so this service starts even with not launching a graphical environment, but why would you want that with a tablet driver?

Reply:
I was originally going to get it to trigger on sockets with udev rules but I found out that the GUI needs the daemon to be running even if a tablet isn't connected. I dumbly replaced `/sockets/default/` instead of removing the patch